typescript/cli/src/context/strategies/signer/MultiProtocolSignerManager.ts

```diff
@@ -48,12 +55,18 @@ export class MultiProtocolSignerManager {
    */
   protected initializeStrategies(): void {
     for (const chain of this.chains) {
-      if (this.multiProvider.getProtocol(chain) !== ProtocolType.Ethereum) {
+      const protocolType = this.multiProvider.getProtocol(chain);
+
+      if (
+        protocolType !== ProtocolType.Ethereum &&
+        protocolType !== ProtocolType.CosmosNative
```

Comment:
Considering that this check is similar to the one below, maybe it would be better to a single function that checks if the current protocol type is supported for initializing a strategy/signer so that we don't have to update this in multiple places, wdyt?